java/flight/src/main/java/org/apache/arrow/flight/DictionaryUtils.java

```diff
@@ -74,4 +70,33 @@ static Schema generateSchemaMessages(final Schema originalSchema, final FlightDe
     }
     return schema;
   }
+
+  /**
+   * Generate the schema to send with flight messages.
```

Comment:
it might be good to describe  a little big what this does with the inputs

Reply:
Sure. I have added more comments to it.